test/blocks/product-card/product-card.test.js

```diff
@@ -6,40 +6,85 @@ const { loadBlock, decorateBlock } = await import('../../../hlx_statics/scripts/
 
 
 const productCardBlock = document.querySelector('div.product-card');
-await decorateBlock(productCardBlock);
+await decorateBlock(productCardBlock); // todo: should figure out why this is required for this block only
```

Comment:
What's the issue here?

Reply:
If there are issues - please make it a little more descriptive in the comment block